src/auxiliary.jl

```diff
@@ -187,9 +231,9 @@ end
 For `a::Taylor1` returns `zero(a[1])` while for `a::TaylorN` returns
 a zero of a k-th order `HomogeneousPolynomial` of proper type.
 """
-zero_korder(a::Taylor1, ::Int) = zero(a[1])
+zero_korder(a::Taylor1, ::Int) = zero(a[0])
 
-zero_korder(a::TaylorN, k::Int) = HomogeneousPolynomial(zero(a[1][1]), k)
+zero_korder(a::TaylorN, k::Int) = HomogeneousPolynomial(zero(a[0][0]), k)
```

Comment:
Should this be `a [0][1]`?

Reply:
Good catch! Changed to `constant_term(a)`. Thanks!